workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -102,14 +88,7 @@ internal static void MenuBuild(BuildEnvironment environment, params string[] wor
             {
                 try
                 {
-                    LocalLaunch.BuildConfig();
-
-                    foreach (var workerType in workerTypes)
-                    {
-                        BuildWorkerForEnvironment(workerType, environment);
-                    }
-
-                    Debug.LogFormat("Completed build for {0} target", environment);
+                    BuildWorkers(workerTypes, environment);
```

Comment:
So I'm not sure when this behaviour got changed (no changelog 🤔), but I think invoking the CLI method to build something which you don't have build support for should fail rather than skip.

Reply:
hm... yeah though that was how it was before this PR.
What we could do: report back whether any builds were skipped, and if yes: throw an exception in the cli

Reply:
Yeah it was a regression introduced in a previous PR. That seems like a reasonable course of action 😄